app/api/groups/get_team_descendants.go

```diff
@@ -0,0 +1,180 @@
+	}
+
+	var result []teamDescendant
+	service.MustNotBeError(query.Scan(&result).Error())
+
+	groupIDs := make([]int64, 0, len(result))
+	resultMap := make(map[int64]*teamDescendant, len(result))
+	for index, groupRow := range result {
+		groupIDs = append(groupIDs, groupRow.ID)
+		resultMap[groupRow.ID] = &result[index]
+		result[index].Members = []teamDescendantMember{}
+	}
+
+	var parentsResult []teamDescendantParent
+	service.MustNotBeError(srv.Store.Groups().
+		Select("parent_links.idGroupChild AS idLinkedGroup, groups.ID, groups.sName").
+		Joins(`
+			JOIN groups_groups AS parent_links ON parent_links.idGroupParent = groups.ID AND
+				parent_links.sType = 'direct' AND parent_links.idGroupChild IN (?)`, groupIDs).
+		Joins(`
+			JOIN groups_ancestors AS parent_ancestors ON parent_ancestors.idGroupChild = parent_links.idGroupParent AND
+				parent_ancestors.idGroupAncestor = ?`, groupID).
+		Order("groups.ID").
+		Scan(&parentsResult).Error())
+
+	for _, parentsRow := range parentsResult {
+		resultMap[parentsRow.LinkedGroupID].Parents = append(resultMap[parentsRow.LinkedGroupID].Parents, parentsRow)
+	}
+
+	var membersResult []teamDescendantMember
+	service.MustNotBeError(srv.Store.Users().
+		Select(`
+			member_links.idGroupParent AS idLinkedGroup,
+			users.idGroupSelf, users.ID, users.sFirstName, users.sLastName, users.sLogin, users.iGrade`).
+		Joins(`
+				JOIN groups_groups AS member_links ON
+					member_links.sType IN ('direct', 'invitationAccepted', 'requestAccepted') AND
+					member_links.idGroupChild = users.idGroupSelf AND
+					member_links.idGroupParent IN (?)`, groupIDs).
+		Order("users.ID").
+		Scan(&membersResult).Error())
+
+	for _, membersRow := range membersResult {
+		resultMap[membersRow.LinkedGroupID].Members = append(resultMap[membersRow.LinkedGroupID].Members, membersRow)
+	}
+
+	render.Respond(w, r, result)
+	return service.NoError
+}
+
+type teamDescendantParent struct {
+	// required:true
+	ID int64 `sql:"column:ID" json:"id,string"`
+	// required:true
+	Name string `sql:"column:sName" json:"name"`
+
+	LinkedGroupID int64 `sql:"column:idLinkedGroup" json:"-"`
+}
+
+type teamDescendantMember struct {
+	// User's `ID`
+	// required:true
+	ID int64 `sql:"column:ID" json:"id,string"`
+	// required:true
+	SelfGroupID int64 `sql:"column:idGroupSelf" json:"self_group_id"`
+	// Nullable
+	FirstName *string `sql:"column:sFirstName" json:"first_name"`
+	// Nullable
+	LastName *string `sql:"column:sLastName" json:"last_name"`
+	// required:true
+	Login string `sql:"column:sLogin" json:"login"`
+	// Nullable
+	Grade *int32 `sql:"column:iGrade" json:"grade"`
+
+	LinkedGroupID int64 `sql:"column:idLinkedGroup" json:"-"`
+}
+
+// swagger:model
+type teamDescendant struct {
+	// The team's `groups.ID`
+	// required:true
+	ID int64 `sql:"column:ID" json:"id,string"`
+	// required:true
+	Name string `sql:"column:sName" json:"name"`
+	// required:true
+	Grade int32 `sql:"column:iGrade" json:"grade"`
+
+	// Team's parent groups
```

Comment:
Add that it is among the input group's descendants.

Reply:
done